### PR TITLE
Set PR pending status when start

### PR DESCRIPTION
**Pull request by beeender**
Also fixed a syntax issue in pr.yml

### CODE REVIEW

**Comment by beeender** [comment]
It is already flied

File: concourse/pipeline/job_def.lib.yml [hunk 1]
```diff
@@ -70,7 +70,9 @@ on_success: #@ trigger["on_success"]
 on_failure: #@ trigger["on_failure"]
 on_error: #@ trigger["on_error"]
 plan:
-- #@ trigger["plan"]
+#@   for trigger_plan in trigger["plans"]:
```
Comment (xuebinsu) [inline concourse/pipeline/job_def.lib.yml:73]:
```suggestion
#@ for trigger_plan in trigger["plans"]:
```

File: concourse/pipeline/job_def.lib.yml [hunk 2]
```diff
@@ -70,7 +70,9 @@ on_success: #@ trigger["on_success"]
...
+- #@ trigger_plan
+#@   end
```
Comment (xuebinsu) [inline concourse/pipeline/job_def.lib.yml:75]:
```suggestion
#@ end
```

**Review by xuebinsu — COMMENTED** [review]
LGTM

File: concourse/pipeline/job_def.lib.yml (hunk 1 above)
Comment (beeender) [reply concourse/pipeline/job_def.lib.yml:73]: No, the indention is on purpose 
```
#@ def build_test_job(param): line 62
...
#@   for trigger_plan in trigger["plans"]:
```

File: concourse/pipeline/job_def.lib.yml (hunk 2 above)
Comment (beeender) [reply concourse/pipeline/job_def.lib.yml:75]: ditto

**Review by xuebinsu — APPROVED** [review]

**Review by Sasasu — APPROVED** [review]

**Pull request merged**